app/services/api/characters/api.fake.ts

```diff
@@ -5,7 +5,10 @@ import { ApiConfig, HydrogenAPI, SagaSauceAPI } from '../IHydrogenAPI'
 import R from 'ramda'
 import AsyncStorage from '@react-native-async-storage/async-storage'
 
-const sortBy = R.sortBy(R.prop('initiative'))
+const sortWith = R.sortWith([
```

Comment:
@JonnoKiwi WELL DONE!!!!!! Great job researching this and finding it!

Reply:
Thanks John, it's satisfying when it comes together  ^^